src/runners/sac.py

```diff
@@ -157,40 +177,43 @@ def eval(self):
 
             # Quickly dump recently-completed episode's experience to the multithread queue,
             # as long as the episode resulted in "success"
-            if self.agent.cfg[
+            if self.exp_config[
                 "record_experience"
             ]:  # and self.metadata['info']['success']:
                 self.file_logger.log("writing experience")
                 self.agent.save_queue.put(experience)
 
-        self.checkpoint_model(ep_ret, self.cfg["max_ep_len"])
+            self.checkpoint_model(ep_ret, j)
+
         self.agent.update_best_pct_complete(info)
 
         return val_ep_rets
 
     ## Sid needs to take care of the actor_critic objects
     def checkpoint_model(self, ep_ret, n_eps):
         if ep_ret > self.best_ret:  # and ep_ret > 100):
-            path_name = f"{self.cfg['model_save_path']}/best_{self.cfg['experiment_name']}_episode_{n_eps}.statedict"
+            path_name = f"{self.runner_config['model_save_dir']}/best_{self.exp_config['experiment_name']}_episode_{n_eps}.statedict"
```

Comment:
Is it intentional that you changed the output path in 2 places and not in others?

Reply:
Yeah the code in checkpoint_model is legacy and not being used anywhere

Reply:
Can we delete it then?